### PR TITLE
Updated upgrade to give more verbose output

### PR DESCRIPTION
**Pull request by waveywaves**
Update upgrade verb to give output same as install verb while installing a new chart.

**What this PR does / why we need it**:
wrt https://github.com/helm/helm/issues/6172

### CODE REVIEW

**Comment by waveywaves** [comment]
cc @hickeyma Can you review this ? 
This is the output the command gives now.
```
➜  helm3test helm upgrade -i trafickd-2 kubernetes-charts/traefik
Release "trafickd-2" does not exist. Installing it now.
NAME: trafickd-2
LAST DEPLOYED: 2019-08-11 15:48:36.555732669 +0530 IST m=+1.741079837
NAMESPACE: default
STATUS: deployed

NOTES:
1. Get Traefik's load balancer IP/hostname:

     NOTE: It may take a few minutes for this to become available.

     You can watch the status by running:

         $ kubectl get svc trafickd-2-traefik --namespace default -w

     Once 'EXTERNAL-IP' is no longer '<pending>':

         $ kubectl describe svc trafickd-2-traefik --namespace default | grep Ingress | awk '{print $3}'

2. Configure DNS records corresponding to Kubernetes ingress resources to point to the load balancer IP/hostname found in step 1      
```

**Comment by bacongobbler** [comment]
Dupe of #6200.

**Comment by bacongobbler** [comment]
Dupe of #6200.

**Comment by bacongobbler** [comment]
Actually, this looks to be the correct behaviour. The output should not be hidden behind the --debug flag.

File: cmd/helm/upgrade.go [hunk 1]
```diff
@@ -90,6 +90,7 @@ func newUpgradeCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 				histClient.Max = 1
 				if _, err := histClient.Run(args[0]); err == driver.ErrReleaseNotFound {
 					fmt.Fprintf(out, "Release %q does not exist. Installing it now.\n", args[0])
+
```
Comment (bacongobbler) [inline cmd/helm/upgrade.go]: Can you remove this newline?

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: cmd/helm/upgrade.go (hunk 1 above)
Comment (waveywaves) [reply cmd/helm/upgrade.go]: Updated. Removed the newline.

**Review by bacongobbler — APPROVED** [review]

**Comment by waveywaves** [comment]
@bacongobbler Thanks for checking this out !

**Pull request merged**